LEGO1/tgl/d3drm/renderer.cpp

```diff
@@ -0,0 +1,323 @@
+#include "impl.h"
+
+using namespace TglImpl;
+
+// OFFSET: LEGO1 0x100a15e0
+Renderer* CreateRenderer()
```

Comment:
This function should be part of the `Tgl` namespace as declared in `tgl.h`?

Reply:
Yup, missed `Tgl::` on there, forgot how I was doing the namespaces.